src/styles.css

```
@@ -216,6 +216,43 @@
 	overflow-wrap: anywhere;
 }
+/* Text that only assistive tech reads. The standard clip-rect technique: unlike
+   `display:none` / `visibility:hidden` it stays in the accessibility tree. */
+.qa-visually-hidden {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	margin: -1px;
+	padding: 0;
+	overflow: hidden;
+	clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)